### PR TITLE
Release 10.10

### PR DESCRIPTION
**Pull request by swethakundaram**

### CODE REVIEW

File: learn/wavemaker-release-notes/v10-10-0.md [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+
+### Run Time Environment
+
+| Description | Version |
+| --- | --- |
+| JDK | 1.8 |
+| WebSphere | 8.5.5 |
+| JBoss | 7.1.1 |
+| Tomcat | 8.5* |
+| Cordova (mobile apps) <br> Android <br> iOS | 10.0.0 <br> 9.0.0  <br> 6.1.0 |
```
Comment (sboyina) [inline learn/wavemaker-release-notes/v10-10-0.md:95]: 9.0.0 has to be updated to 10.1.0

**Review by sboyina — CHANGES_REQUESTED** [review]

File: learn/wavemaker-release-notes/v10-10-0.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+
+## New Features
+
+---
+
+### Prefab Localization
+
+#### Override Prefab locale messages in the application
+
+You can override the values for the Prefab local keys by using the "prefabMessages" object in the I18N folder.
```
Comment (vibhus) [inline learn/wavemaker-release-notes/v10-10-0.md:16]: local --> "locale"

File: learn/wavemaker-release-notes/v10-10-0.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
+#### Override Prefab locale messages in the application
+
+You can override the values for the Prefab local keys by using the "prefabMessages" object in the I18N folder. 
+
+- For example, a Prefab contains languages including English, Spanish, and Arabic. To edit a prefab message key of Spanish, you override the message keys in the `es.json` file where the prefab messages object is provided, placed in the I18N folder. Similarly, to edit Arabic, you edit the message keys in the `ar.json` file.
+
+#### Handling unsupported Prefab locale
+
+- Suppose the application supports a locale that Prefab does not support. In that case, the prefab messages are picked from the Prefabs' en.json file for that specific locale in the output folder instead of empty messages.
```
Comment (vibhus) [inline learn/wavemaker-release-notes/v10-10-0.md:22]: Also mention, "Even though the locale is not supported in the prefab, you can still provide localized messages in that language by extending the language.json file and adding keys manually." E.g. if prefab supports only English and Spanish but app supports English, Spanish and Arabic, you can add the respective locale message keys in the prefabMessages object inside the ar.json file.

File: learn/wavemaker-release-notes/v10-10-0.md [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+
+- For example, a Prefab contains languages including English, Spanish, and Arabic. To edit a prefab message key of Spanish, you override the message keys in the `es.json` file where the prefab messages object is provided, placed in the I18N folder. Similarly, to edit Arabic, you edit the message keys in the `ar.json` file.
+
+#### Handling unsupported Prefab locale
+
+- Suppose the application supports a locale that Prefab does not support. In that case, the prefab messages are picked from the Prefabs' en.json file for that specific locale in the output folder instead of empty messages. 
+
+#### Merging App and Prefab locale keys
+
+- The generated output folder contains overridden localization keys. You can merge the Prefab messages configured in app locale files and the Prefab locale files.
```
Comment (vibhus) [inline learn/wavemaker-release-notes/v10-10-0.md:26]: This may not be required.

**Review by vibhus — CHANGES_REQUESTED** [review]

**Review by sboyina — APPROVED** [review]

**Review by vibhus — APPROVED** [review]

**Pull request merged**